scripts/simulate_live_data_stress.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2024 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+
```

Comment:
Could we move this whole script into the simulate live data utility script where if no source is specified, we use your implementation to draw new images. This way, we won't grow the number of separate utility scripts we have and keep any live view testing/simulation within one script to keep things simple.

Reply:
I've dropped this out of the PR and I'll do a separate PR with the merged script.

Reply:
Thanks, I'll give this another quick review now